[bug] Bumping to ^v1.52.7 or ^v1.52.8 results in a bunch of rule definitions not being found

### Describe the bug

The errors I am seeing on upgrade from v1.52.4 -> 8 are:

```
1:1  error  Definition for rule '@eslint-react/hooks-extra/no-unnecessary-use-prefix' was not found             @eslint-react/hooks-extra/no-unnecessary-use-prefix
  1:1  error  Definition for rule '@eslint-react/hooks-extra/prefer-use-state-lazy-initialization' was not found  @eslint-react/hooks-extra/prefer-use-state-lazy-initialization
```

I believe this was introduced in v1.52.7 as when I force down to v.1.52.6 lint seems to work fine without any other changes.

For reference I am extending: `'plugin:@eslint-react/recommended-type-checked-legacy',` since I am on on eslint v8.

### Reproduction

Not much I have I can share at this point, but it's a bump from v1.52.4 -> v1.52.8.

Here's a snippet of my `.eslintrc.json`

```
            files: ['**/*.ts', '**/*.tsx'],
            plugins: [
                'import',
                'unused-imports',
            ],
            extends: [
                'plugin:@eslint-react/recommended-type-checked-legacy',
            ],
```

### Expected behavior

No errors as part of a patch upgrade.

### Platform and versions

```Text
node v22.0.0

"@eslint-react/eslint-plugin": "^v1.52.8",
```

### Stack trace

```Text

```

### Additional context

_No response_

--- a/apps/website/package.json
+++ b/apps/website/package.json
@@ -36,7 +36,7 @@
     "@eslint/js": "^9.34.0",
     "@eslint/markdown": "^7.2.0",
     "@local/configs": "workspace:*",
-    "@mdx-js/mdx": "^3.1.0",
+    "@mdx-js/mdx": "^3.1.1",
     "@tailwindcss/postcss": "^4.1.12",
     "@theguild/remark-mermaid": "^0.3.0",
     "@tsconfig/next": "^2.0.3",

--- a/packages/plugins/eslint-plugin/src/index.ts
+++ b/packages/plugins/eslint-plugin/src/index.ts
@@ -38,7 +38,7 @@ const plugin: CompatiblePlugin = {
     ...react.rules,
     ...padKeysLeft(reactDom.rules, "dom/"),
     ...padKeysLeft(reactWebApi.rules, "web-api/"),
-    ...padKeysLeft(reactHooksExtra, "hooks-extra/"),
+    ...padKeysLeft(reactHooksExtra.rules, "hooks-extra/"),
     ...padKeysLeft(reactNamingConvention.rules, "naming-convention/"),
     ...padKeysLeft(reactDebug.rules, "debug/"),
   },
@@ -71,6 +71,7 @@ export default {
     ["recommended-typescript-legacy"]: toLegacyConfig(recommendedTypeScriptConfig),
     ["x"]: xConfig,
     ["x-legacy"]: toLegacyConfig(xConfig),
+
     // Part: deprecated presets
     /** @deprecated Use `x` instead */
     ["core"]: xConfig,

